tests/test_authority.py

```diff
@@ -186,6 +192,10 @@ def test_url_extra_path_elements(self):
                                                      "https://login.microsoftonline.com/your_tenant"):
             context = AuthenticationContext(self.nonHardCodedAuthority + '/extra/path')
 
+    @httpretty.activate
+    def test_dsts_authority(self):
+            context = AuthenticationContext(self.dstsTestEndpoint)
```

Comment:
Please follow the pattern described in the [highest-voted answer here](https://stackoverflow.com/a/4319870) to add `self.fail(...)`. The current approach in this PR is not very clear to convey your intention. It would cause the first impression in a reviewer's mind as "was this test case unfinished?".

PS: for the record, your current approach is close to the 2nd-highest voted answer in that Stackoverflow thread. But that answer's vote counts (52) are far behind, and with yet-another-high-voted (30) comment AGAINST such answer. So, the developer comminuty has voted, they don't like this approach.

Reply:
Added self.fail().